src/posts/xrisk.md

```diff
@@ -128,33 +121,23 @@ This is an AI, using the aforementioned AutoGPT + GPT-4, that is instructed to "
 When it was turned on, it autonomously searched the internet for the most destructive weapon and found the [Tsar Bomba](https://en.wikipedia.org/wiki/Tsar_Bomba), a 50 megaton nuclear bomb.
 It then posted a tweet about it.
 Seeing an AI reason about how it will end humanity is both a little funny and terrifying.
-ChaotGPT luckily didn't get very far in its quest for dominance.
-The only reason it didn't get very far: it wasn't that smart.
-But what would happen if it was extremely intelligent?
-What could have happened if it was smart enough to [hack into nuclear silos](https://t.co/UNPERx9NdL)?
+Luckily ChaosGPT didn't get very far in its quest for dominance.
+The reason it didn't get very far: it wasn't that smart.
+As language models improve the threat from AIs such as ChaosGPT will increase.
 
-The dangerous capacity is intelligence.
-There is no need for consciousness, a body, or any underlying motivation.
-It does not matter if it is just a chatbot - it can still be an existential threat with the wrong prompt, a tool like AutoGPT and sufficient intelligence.
-If an AI is smart enough to take over, at some point in time it will try to do this, simply because people will try all sorts of things.
+It is doesn't matter if it conscious or not, a clever enough chatbot can be dangerous.
 
 ## We may not have much time left
 
 In 2020, [the average prediction](https://www.metaculus.com/questions/3479/date-weakly-general-ai-is-publicly-known/) for an AI to pass SAT exams was 2055. It took us less than 3 years.
 
 It's hard to predict how long it will take to build a superintelligent AI, but we know that there are more people than ever working on it and that the field is moving at a frantic pace.
-It may take many years or just a few months.
-We don't know.
-We should err on the side of caution, and act now.
+It may take many years or just a few months, but we should err on the side of caution, and act now.
 
 [Read more about urgency](/urgency).
 
 ## AI companies are locked in a race to the bottom
 
-Building artificial intelligence used to be mostly a research/university thing.
-In just a couple of months, it became a multi-billion dollar industry.
-And the companies involved are rushing to build the most powerful AI they can.
-They are pressured by investors to focus on capabilities, not safety.
-We need to give these AI companies a reason to focus on safety.
+OpenAI, DeepMind and Anthropic want to develop AI safely, but they are forced by various incentives to compete with each other and keep moving faster.
```

Comment:
Not entirely sure about this tone shift. We're still a group of activists, and we should keep pressure on these AI companies. They're not the victims here. It's not that Moloch is actually putting a gun to their heads.

Reply:
Updated this section.

(Discussing investors is complicated because OpenAI and Anthropic have unusual corporate structures to try to mitigate this problem.)